### ISSUE TITLE
3. Capacity to delete or add rows or columns

### ISSUE THREAD

**Issue by broadwell** [sul-cidr/scriptchart#10]

**Comment by broadwell** [comment]
Out of the box, reactabular enables adding/removing columns (by toggling their visibility) and removing **but not re-adding** rows, (by actually deleting their contents from the table). Providing full row-toggling capabilities, if desired, would require some custom code but seems feasible.
This does however raise a UX question regarding how such table faceting options  relate to the main selection form -- should the form always reflect the status of the table (e.g., indicating which manuscripts and letters are currently available and/or visible), and if not, how do we make this distinction clear in the UI?

**Comment by csbailey5t** [comment]
We've talked through _not_ adding some of this capacity in favor of just having the user re-submit the form with the particular options they want. I could see how having a specific graphical capacity to delete might still be useful but I think it would count as a stretch feature. 

If we kept it so that you just used the form to manage what is included in the table, we wouldn't need to worry about syncing the form to the table otherwise.

**Comment by broadwell** [comment]
Yeah that seems like a good way to conceptualize it -- the form determines the table content, so the table features would just provide some manipulation of how the content is presented. Within that framework, though, it's probably better to facilitate sorting by shelfmark or date via the table rather than the form, assuming we're just talking about the kinds of vanilla alphanumeric sorts that Reactabular supports out of the box.
